File: Spring/Spring-Core-Shop-Anootaions/src/main/java/com/Employee.java
package com;

import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Service;

@Service(value="emp")
public class Employee {

	private int empId=10;
	private String empName="Praveen";
	
	@Autowired
	private Department dept;
	

	public Department getDept() {
		return dept;
	}

	public void setDept(Department dept) {
		this.dept = dept;
	}

	public int getEmpId() {
		return empId;
	}

	public void setEmpId(int empId) {
		this.empId = empId;
	}

	public String getEmpName() {
		return empName;
	}

	public void setEmpName(String empName) {
		this.empName = empName;
	}

}

File: Angular/ReactiveForms/src/app/employee.ts
export class Employee {
    empId:number;
    empName:string;
    salary:number;

    getEmpId():number{

        return this.empId;
    }
    getEmpName():string{
        return this.empName;
    }

    getSalary():number{
        return this.salary;
    }
}
